### PR TITLE
Removed event-stream dependency

### PR DESCRIPTION
**Pull request by aal89**
**What does this PR do?**

~~Locks~~ Removes the event-stream dependency and updates nodemon dependency to `1.18.7` to mitigate a possible attack. Also included the `package-lock.json` file to have [reproducible builds](https://reproducible-builds.org/). This most probably helps to stay safer.

**Background**
https://github.com/dominictarr/event-stream/issues/116

**Resolves**
https://github.com/JacksonGariety/gulp-nodemon/issues/163

### CODE REVIEW

**Comment by TwoAbove** [comment]
https://github.com/JacksonGariety/gulp-nodemon/pull/165
This one is better imo

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   "homepage": "https://github.com/JacksonGariety/gulp-nodemon",
   "dependencies": {
     "colors": "^1.2.1",
-    "event-stream": "^3.3.4",
+    "event-stream": "3.3.4",
```
Comment (yoelfme) [inline package.json]: maybe add the `=` operator to be more explicit

**Comment by joebowbeer** [comment]
Or update to event-stream@4.0.1, which is also clean of the bogus flatmap-stream dep.
Better yet, by the burn-me-once principle: move off of event-stream?

**Comment by aal89** [comment]
I have updated the PR. Ever since @joebowbeer commented: 'burn-me-once principle: move off of event-stream', I checked if it was used anywhere and ran the test, but couldn't find any problems after removing it entirely. Could people verify? Then I think this is the best fix? Don't require it if you don't need it.

**Comment by rkmarks** [comment]
I was curious why `event-stream` is a dependency, glad this is being removed in this PR. 👍

**Comment by aal89** [comment]
@rkmarks I looked through the history and it seems that it once had a purpose, however somewhere down the road it had been removed, just never as a dependency.

`map-stream` was being replaced by `event-stream` [here](https://github.com/JacksonGariety/gulp-nodemon/commit/4fe7d5cae64ab93cc7ee506d78a0655f10265c78#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) and actually using a tilde for version selection. It later got updated and changed to a carot for version selection [here](https://github.com/JacksonGariety/gulp-nodemon/commit/8de5f6664abb0ab765e9e406dc1671cf482c5b96#diff-b9cfc7f2cdf78a7f4b91a753d10865a2). Continuing down the road it got updated, eventually requiring the dirty version. (disc: not implying the tilde/carot thing is bad).

**Comment by yoelfme** [comment]
that's all @aal89, thanks for your help. @JacksonGariety woulb be great if we can get these changes in a new version

**Pull request merged**